final_task/README.md

```diff
@@ -1,35 +1,53 @@
 # RSS_READER
 ---------------------------------------------------------------------------
-RSS reader is a command-line utility.
+RSS reader is a command-line utility.  
 
 ### Usage
 ---------------------------------------------------------------------------
-usage: rss_reader.py source [-h] [--version] [--json] [--verbose] [--limit LIMIT]  
+usage: rss_reader.py [-h] [--source SOURCE] [--version] [--json] [--verbose]  
+                     [--limit LIMIT] [--date DATE]  
 
 Pure Python command-line RSS reader.  
-positional arguments:  
-source RSS URL  
+
 optional arguments:  
-    -h, --help show this help message and exit  
-    --version Print version info  
-    --json Print result as JSON in stdout  
-    --verbose Outputs verbose status messages  
-    --limit LIMIT Limit news topics if this parameter provided  
+ - -h, --help&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;*show this help message and exit*  
+ - --source SOURCE&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;*RSS URL*  
+ - --version&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;*Print version info*  
+ - --json&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;*Print result as JSON in stdout*  
+ - --verbose&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;*Outputs verbose status messages*  
+ - --limit LIMIT&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;*Limit news topics if this parameter provided*  
+ - --date DATE&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;*News from the specified day will be printed out. Format: YYYYMMDD*  
```

Comment:
It is better to use some *markdown* language features to do it.

Reply:
fixed